code/how-to-collapse-borders/src/bin/problem.rs

```diff
@@ -19,6 +19,8 @@ struct Term {
 }
 
 fn main() -> Result<()> {
+    color_eyre::install()?;
+
```

Comment:
Is this necessary? My understanding was we only needed `color_eyre::install()` if it want to set up panic hooks.

I'd remove it from this because we have other sections on how to set up panic hooks that work well with terminal user interfaces, and installing it twice can cause errors (e.g. if a user copy pastes this code but already has a panic hook set up).

Reply:
@kdheepak I noticed no build errors were popping up, so I'll take it out soon.